_posts/2013-02-19-background-image-options.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: background-image-options
 url: background-img-options
-prefixed: false
+prefixed: true
```

Comment:
I think we should still keep `prefixed: false` for `background-image-options`.
- It's *only* the *not-yet-standard* value of `text` that needs a prefix
- ..and *ony* in WebKit/Edge

Reply:
From a user point of view, if I see `no prefixes!` on a property/group of properties, I wouldn't bother expanding the post and would just trust the website and use no prefixes. So for that reason, I think it would be misleading to mark it as no prefixes needed.

Though I understand that the text value is not standard and you might not want to mark the whole group as prefixed.

Ideally, a third type like `partial prefix` or something similar to that could be more appropriate, but since that's not an option, I think marking it as prefixed would encourage the users to expand the post and read the details.

But in the end, it's your call. If you believe no prefixes is the way to go, I will revert that change back to `prefixed: false`.

Reply:
I guess you're right. Ideally we'd have a `partially` label, but since we do not I guess the 'danger' of not-prefixing "text" is greater than unnecessarily prefixing e.g. `background-size: cover;`.

LGTM 🤘🏻